mobileapp/lib/enum/selectedType.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:mobileapp/domain/managers/binding_manager.dart';
+import 'package:mobileapp/domain/model/Binding.dart';
+
+enum SelectedType {
+  CINEMA,
+  MUSEUM,
+}
+
+extension SelectedTypeExtension on SelectedType {
+  String get typeId {
+    switch (this) {
+      case SelectedType.CINEMA:
+        return 'Q41253';
+      case SelectedType.MUSEUM:
+        return 'Q33506';
+    }
```

Comment:
Why just these two?

Reply:
I just put these two because other people can want to contribute and it's an easy way to work with flutter, however, if you prefer I can put all of them :)